CHANGELOG

```diff
@@ -1,3 +1,9 @@
+0.5.2 (2019-10-16)
+------------------
+
+* Fix logging error when command finish with allowed exitstatus (bsc#1153749)
```

Comment:
```suggestion
* Do not log the exit statuses listed in allowed_exitstatus (bsc#1153749)
```

Reply:
It is still logged as seen below. Just with info instead of error. I redefine success? method. So it report success also for non-zero exit codes if it is allowed in parameters.